tests/muxcable_test.py

```diff
@@ -416,6 +421,84 @@ def test_config_muxcable_tabular_port_with_incorrect_port(self):
 
         assert(result.exit_code == 1)
 
+    def test_show_muxcable_eye_info(self):
+        runner = CliRunner()
+        db = Db()
+
+        with mock.patch('os.geteuid') as patched_util_outer:
```

Comment:
FYI, you might be able to save some indentation by using `@mock.patch(...)` decorators in some places rather than `with mock.patch(...)` statements. It depends on if you want the patch to apply to the entire function or not. This can be saved for a future cleanup/refactor PR, just thought I'd give an FYI.

Reply:
fixed